packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -708,17 +708,28 @@ export class GarbageCollector implements IGarbageCollector {
 			return;
 		}
 
+		// If sweep is disabled, we'll tombstone both tombstone-ready and sweep-ready nodes.
+		const { toTombstone, toDelete } = this.configs.shouldRunSweep
```

Comment:
```suggestion
		const { nodesToTombstone, nodesToDelete } = this.configs.shouldRunSweep
```

toTombstone / toSweep doesn't really tell that its a list of nodes that are ready to be tombstoned.